src/index.ts

```diff
@@ -23,12 +23,22 @@ async function run(): Promise<void> {
     const pullNumber = pullRequest.data.number;
     const htmlUrl = pullRequest.data.html_url;
 
-    if (reviewers.length > 0) {
+    if (reviewers.length > 0 || team_reviewers.length > 0) {
```

Comment:
Did you test that this is the required logic?

Reply:
I am currently using my fork in workflows and it works for both sets of reviewers.
Would love to switch back.

Reply:
Cool, thanks for the confirmation :)